Dockerfile

```diff
@@ -114,4 +114,3 @@ WORKDIR /home/dnanexus
 RUN ["chmod", "+x", "parliament2.py"]
 RUN ["chmod", "+x", "parliament2.sh"]
 
-ENTRYPOINT ["python","/home/dnanexus/parliament2.py"]
```

Comment:
@edawson Please merge master back into this branch (hopefully the testing framework will work this time) so that we can confirm how this line affects the Docker call (if at all). Thanks!